GVRf/Framework/framework/src/main/AndroidManifest.xml

```diff
@@ -25,9 +25,12 @@
         android:minSdkVersion="19"
         android:targetSdkVersion="19" />
 
+    <uses-permission android:name="com.samsung.android.hmt.permission.READ_SETTINGS" />
```

Comment:
If it is required by all the backends then it is fine. We could revisit it when we work on adding more backends in the future.